smtp.go

```diff
@@ -375,11 +376,11 @@ func SendMail(addr string, a smtp.Auth, from string, to []string, msg []byte) er
 			}
 		}
```

Comment:
We should check here if scheme is starttls:// but server did not announce STARTTLS extension => raise an error

Reply:
Good call, I've added the error

Reply:
This diff looks a bit strange on github (don't know why) but I will have a look at that after the merge. Thanks!